examples/python/2_large_model/verify_235b_direct_load.py

```
@@ -0,0 +1,595 @@
+# Memory monitoring
+# ---------------------------------------------------------------------------
+
+def get_rss_gb():
+    """Current RSS (not peak) in GB, read from /proc/self/status."""
+    try:
+        with open("/proc/self/status") as f:
+            for line in f:
+                if line.startswith("VmRSS:"):
+                    return int(line.split()[1]) / (1024 * 1024)  # kB → GB
+    except Exception:
+        pass
```

Ruff (0.15.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)